impeller/renderer/backend/gles/buffer_bindings_gles.cc

```diff
@@ -16,6 +16,23 @@
 
 namespace impeller {
 
+namespace {
+class ScopedBytes {
+ public:
+  void Reset(size_t size) {
+    bytes_ = std::make_unique<uint8_t[]>(size);
```

Comment:
why not use malloc/free?

Reply:
This is using new[] and delete[], essentially the same thing.  The difference here is that there is no initialization happening when the allocated buffer, unlike std::vector::resize.